### ISSUE TITLE
Microsoft.Azure.ActiveDirectory.GraphClient NuGet package requires incorrect version of Microsoft.Data.Services.Client dependency

### ISSUE THREAD

**Issue by bergmeister** [microsoftgraph/msgraph-sdk-dotnet#371]
I am hitting the following issue that has been extensively documented on stackoverflow whereby the NuGet package requires version 5.6.4 of Microsoft.Data.Services.Client but the `Microsoft.Azure.ActiveDirectory.GraphClient` dll actually references `4.6.3`. I therefore have to put in binding redirects to work around this problem
https://stackoverflow.com/questions/45137756/microsoft-azure-activedirectory-graphclient-requires-incorrect-version-of-depend

I am using the latest version (`2.1.1`) of Microsoft.Azure.ActiveDirectory.GraphClien

**Comment by MIchaelMainer** [comment]
This repo is for the Microsoft.Graph client, not the deemphasized ActiveDirectory.Graph client that you mention. The ActiveDirectory.Graph client hasn't been updated in over two years. The general guidance is to use the Microsoft.Graph client. There are some edge cases that aren't yet implemented in Microsoft Graph but that are implemented the deemphasized ActiveDirectory Graph. In those cases, we suggest you make calls directly against the endpoint.

**Comment by bergmeister** [comment]
@MIchaelMainer Thanks for the informative answer. This is very useful to know. NuGet is planning to add tags such as e.g. `Legacy` to NuGet packages soon, maybe this could be applied to the AD graph client package then?
I am happy with this issue being closed and as a small thank you for your time. I opened a small PR where I converted 2 old projects to use packagereference and update nuget packages of test projects
